lisa/features/startstop.py

```diff
@@ -42,3 +42,6 @@ def restart(self, wait: bool = True) -> None:
         self._log.info("restarting")
         self._restart(wait=wait)
         self._node.close()
+
+    def retrive_ip(self) -> str:
```

Comment:
It doesn't need the new API. Not all platform will change the IP. Just update the IP in start method of Azure.